cookbook/flytetester.json

```diff
@@ -0,0 +1,65 @@
+ [{
```

Comment:
just so we don't confuse the name with the other flytetester... maybe name it `flyte_tests_manifest.json` or something?